doc/cephadm/operations.rst

```diff
@@ -395,25 +395,34 @@ process is active within the cluster.*
 CEPHADM_CHECK_KERNEL_VERSION
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 The OS kernel version (maj.min) is checked for consistency across the hosts.
-The kernel version of the majority of the hosts is used as the basis for 
+The kernel version of the majority of the hosts is used as the basis for
```

Comment:
Does the command analyze the results and identify anomalies, or are we saying that the *operator/admin* can use the results that way?

Reply:
Not sure about this part, I've just removed the spurious extra space.